### PR TITLE
 validate contact form

### PR DESCRIPTION
**Pull request by Melaku05**
**Creating validate contact form for the portfolio project**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -226,21 +232,46 @@ Array.from(portfolio.children).forEach((item, index) => {
...
+      });
+    }
+  );
 });
+
+// form validation
+const form = document.querySelector('.form');
+const emailInput = form.elements.email;
+
+const INPUT_LOWERCASE = 'Error : Please enter a lowercase input';
```
Comment (Whoistolu) [inline index.js:259]: - [ ] [REQUIRED] You have done a great job so far. however, one of the requirements states that: when a user submits the contact form with the email field containing uppercase characters an error message should show up. I have tested the form but it still submits successfully while inputting uppercase in the email field. Kindly fix this in order to meet up the requirement.
- Also, ensure that the error message shows up close to the `submit` button.
- I see that you have implemented this `function validateEmail(input, invalidLowercase) {
  if (input.value === input.value.toLowerCase()) {
    return true;
  }
  return showError(input, invalidLowercase);
}`
 Which is good. You then need to loop through each letter of the email string to ensure all letters do not allow an uppercase.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Melaku05 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (Melaku05) [reply index.js:259]: Thank you, I did that

**Review by codepantha — APPROVED** [review]
Hi @Melaku05 👋, 

I must commend you for implementing the change requested by the previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**